nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -47,4 +47,7 @@ In addition to numerous new and upgraded packages, this release has the followin
 * A new module was added for the Saleae Logic device family, providing the options `hardware.saleae-logic.enable` and `hardware.saleae-logic.package`.
 * Matrix Synapse now requires entries in the `state_group_edges` table to be unique, in order to prevent accidentally introducing duplicate information (for example, because a database backup was restored multiple times). If your Synapse database already has duplicate rows in this table, this could fail with an error and require manual remediation.
 
+- memtest86+ was updated from 5.00-coreboot-002 to 6.00-beta2, and is now from upstream instead of coreboot as coreboot's 
```

Comment:
Added at the bottom instead of the middle as the other lines need changed anyway due to wrong style.

Opened #176052